### ISSUE TITLE
recording: "is filled by" missing inverse relationship

### ISSUE THREAD

**Issue by luizaogs** [arch-kiosk/arch-kiosk-office#2657]
I just noticed this in testing stuff for ELRI but I had already noted it for MKAP in #2309 so I create a new ticket because it seems to make more sense to fix it for all projects?

<img width="1034" alt="Screenshot 2024-04-25 at 3 15 18 PM" src="https://github.com/arch-kiosk/arch-kiosk-office/assets/67970402/dec82e6a-989b-440c-b583-fee32961acc5">

**Comment by luizaogs** [comment]
@lbestock oops forgot to add you. Maybe something to change in ELRI manually regardless?

**Comment by urapadmin** [comment]
this has to be changed manually in all existing projects. Only for future projects I can add change this in the template.

**Comment by luizaogs** [comment]
Ok. Well I changed MKAP at the time so that one doesn't need this, then.

**Comment by luizaogs** [comment]
- [ ] MADP
- [ ] MKAP
- [ ] BUAP
- [ ] ARCH1900
- [ ] KHPP
- [ ] KAMBE
- [ ] PASU
- [ ] PACHA
- [ ] USTP
- [ ] URAP
- [ ] ELRI

**Comment by luizaogs** [comment]
I did all the ones I could. I don't have logins for ARCH1900, KHPP (where I seem to have a login but not enough privileges to go to the hub), USTP (have a login but can't create a new dock and didn't want to use someone else's).

PASU has a weird warning currently (see #2663 ) so I've left it alone for now. See also #2665 for PACHA.

KAMBE still needs to be synchronized, see #2664.

**Comment by urapadmin** [comment]
@lbestock @luizaogs 
admin passwords for Beset should now accessible for you in the wiki in a page "field projects and users". You can search for it.
Only we three have currently access to that page.

**Comment by urapadmin** [comment]
> I did all the ones I could. I don't have logins for ARCH1900, KHPP (where I seem to have a login but not enough privileges to go to the hub), USTP (have a login but can't create a new dock and didn't want to use someone else's).
> 
I do KHPP and ARCH1900. KHPP does not work properly, currently.

**Comment by luizaogs** [comment]
Ok. With those passwords I can now do USTP.

**Comment by urapadmin** [comment]
you failed because we were in each other's hair, or rather in each other's FileMaker. I have cleared that situation now and leave USTP happily to you. And would you pleeease check that I did the right think in khpp. I am so dangerous with this Covid brain ...

**Comment by lbestock** [comment]
It makes me happy to see you both even in each other's hair while I comment on seminar drafts (the last one was really good, go @maychu16 ).

**Comment by luizaogs** [comment]
Argh are you still in KHPP Lutz? I am also getting an error there right now.

**Comment by luizaogs** [comment]
And now I can't import USTP. I think I will get out of your hair :)

**Comment by urapadmin** [comment]
no, I wasn't. It is FileMaker. It froze again. So I restart Beset entirely. Perhaps that helps. 
I hate FileMaker.

**Comment by urapadmin** [comment]
this will take a bit until everything is up and smooth again...

**Comment by luizaogs** [comment]
KHPP looks fine and I've now done PASU, so we're only missing ARCH1900.

**Comment by urapadmin** [comment]
okay. So you are done? I had to actively monitor Beset while you were doing all of this because that kind of thing (fastly switching between projects and their FileMaker tasks) is what kicks Beset our of balance. Reminds me not for the first time that this is a local server system and running field projects online not a good idea with it. 
Did you get logged out all the time, too? I just had to log in over and over again when switching between projects in the browser.

**Comment by luizaogs** [comment]
Yeah, I'm done. I got a few "your session has timed out" messages but refreshing worked, I didn't have to log in again.
